CMakeLists.txt

```diff
@@ -10,8 +10,8 @@ option(FAASM_OPENMP_TRACING "Trace OpenMP forks in Redis" OFF)
 option(FAASM_PERF_PROFILING "Turn on profiling features as described in debugging.md" OFF)
 
 # SGX functionality
-option(FAASM_SGX_SUPPORT "Add support for SGX" OFF)
-option(FAASM_SGX_SIM_MODE "SGX sim mode flag" OFF)
+set(FAASM_SGX_SUPPORT 0) # Enable/Disable SGX Support
```

Comment:
All on/ off switches should be expressed as `option`s. This way people can pass them as arguments on the command line when configuring, rather than having to change this file. 

Importantly they can then run `cmake -LH` which lists all the project options and their associated help text. If things are set as variables they don't show in this list.

Reply:
Sounds good for me.